<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DungeonSearchDialog</class>
 <widget class="QDialog">
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="searchTypeComboBox">
        <item>
         <property name="text">
          <string>Tile</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Subtile</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tile protection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monster protection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Object protection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Room</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monster</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Object</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Item</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Index:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWidget" name="searchIndexLineEdit">
        <property name="charWidth">
         <number>8</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="searchSpecCheckBox">
        <property name="toolTip">
         <string>Index of a special monster</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Area:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="LineEditWidget" name="searchXLineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>X-coordinate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel">
          <property name="minimumWidth">
           <number>8</number>
          </property>
          <property name="maximumWidth">
           <number>8</number>
          </property>
          <property name="text">
           <string notr="true">:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEditWidget" name="searchYLineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>Y-coordinate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="LineEditWidget" name="searchWLineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>Width</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel">
          <property name="minimumWidth">
           <number>8</number>
          </property>
          <property name="maximumWidth">
           <number>8</number>
          </property>
          <property name="text">
           <string notr="true">,</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEditWidget" name="searchHLineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>Height</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Scroll:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="scrollToCheckBox">
        <property name="toolTip">
         <string>Scroll to the first match</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchCancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="placementButtonGroup"/>
 </buttongroups>
</ui>
